mods/ts/sequences/structures.yaml

```diff
@@ -1163,7 +1170,7 @@ namisl:
 	make: ntmislmk
 		Length: 18
 		ShadowStart: 18
-	active: ntmisl_a
+	active: ntmisl_a # TODO: This is an overlay
```

Comment:
No, because it's not unused. It's just treated like an animation rather than an overlay at the moment. I can try to make the comment clearer on that, though.